src/type.ts

```diff
@@ -15,7 +15,6 @@ export interface Icon {
      * The html to use.
      */
     ligature?: string;
-    icon?: string;
```

Comment:
This is still there no?

Reply:
It was never used in the first place. So I removed it from the type definition.

Reply:
You're right, this must have been a mistake